mne/channels/data/montages/artinis-octamon.txt

```diff
@@ -0,0 +1,35 @@
+MNI Coordinates
```

Comment:
Once we are sure the elc file is correct we can delete these. Must be done before merge.